### ISSUE TITLE
build on windows

### ISSUE THREAD

**Issue by shuZro** [stevenlovegrove/Pangolin#425]
hey steven,

I was wondering what the easiest way would be to build this on windows. I have been trying for a while but can't get it to work.

In the end I would like to import to python

Thanks!

**Comment by stevenlovegrove** [comment]
Hi - Pangolin should build pretty reliably on Windows with a recent Visual Studio and cmake. There are also pre-build binaries available (https://ci.appveyor.com/project/stevenlovegrove/pangolin/builds/19392194/artifacts). I have not tested the python bindings on Windows, so I can't say how easy or difficult that will be to get going.

**Comment by Gatunox** [comment]
shuZro:

Do it as follows,

first create a folder with **NO** spaces on the name. in my case I have a **c:\dev**.
excute from **cmd**, you need to be **admin**, so make sure to excute it as an administrator.

within cmd position yourself in your folder, in my case **c:\dev**.
execute:

    git clone https://github.com/stevenlovegrove/Pangolin.git
    cd Pangolin

then if you plan to use **Python** exeute. ** make sure to be inside pangolin folder**
execute:

    python -mpip install numpy pyopengl Pillow pybind11 **(These were OK)**
    git submodule init && git submodule update **(I noticed that once I delete all i had to donload it again)**

then execute:

    mkdir build
    cd build
    cmake ..

verify the results of CMake generation, and check for the lines showed bellow (I'm running python 2.7 btw), versions could change if you are under 3.7. The mots importat ones are the last two entries I am showing here.

   -- Found PythonInterp: C:/Python27/python.exe (found version "2.7.14")
   -- Found PythonLibs: C:/Python27/libs/Python27.lib
   -- pybind11 v2.2.1
   -- Python Found and Enabled (with pybind11)

Then the more hidden part
You need to edit the **C:\dev\Pangolin\src\CMakeLists.txt*** file.

look for the text **WIN** , since the original CMake is ignoring python compile for windows platforms. you shold find something like the lines copied below.

    option(BUILD_PANGOLIN_PYTHON "Build support for Pangolin Interactive Console" ON)
    if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)
    if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../external/pybind11/CMakeLists.txt")
    add_subdirectory("../external/pybind11" "${CMAKE_CURRENT_BINARY_DIR}/external/pybind11")
    set( pybind11_FOUND true)
    else()
    find_package(pybind11 QUIET)
    endif()

delete the **AND NOT WIN** part from the if clause **if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)**, then save the file.

Finally, execute from **c:\dev\Pangolin\build\**

    cmake --build .

check the results, if you have warnigns **ignore** them, if you have error you need to solve them. In my case the error I got **Fatal error LNK1104: cannot open file 'python27_d.lib**, this is because CMake is compiling the project in debug mode, No clue why, I am no expert in CMAKE. so.

What I did was opening the **C:\dev\Pangolin\build\pangolin.sln** solution on Visual Studio 2017, change solution configuration form **debug** to **release** and recompiled within VS2017.

FINALLY I got all compiled successfully.

    ========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

I hope it helps, keep me posted if you need further asistance.

Regards

**Comment by Gatunox** [comment]
shuZro:

Do it as follows,

first create a folder with NO spaces on the name. in my case I have a c:\dev.
excute from cmd, you need to be admin, so make sure to excute it as an administrator.

within cmd position yourself in your folder, in my case c:\dev.
execute:

git clone https://github.com/stevenlovegrove/Pangolin.git
cd Pangolin

then if you plan to use Python exeute. make sure to be inside pangolin folder
execute:

python -mpip install numpy pyopengl Pillow pybind11 **(These were OK)**
git submodule init && git submodule update **(I noticed that once I delete all i had to donload it again)**

then execute:

mkdir build
cd build
cmake ..

verify the results of CMake generation, and check for the lines showed bellow (I'm running python 2.7 btw), versions could change if you are under 3.7. The mots importat ones are the last two entries I am showing here.

-- Found PythonInterp: C:/Python27/python.exe (found version "2.7.14")
-- Found PythonLibs: C:/Python27/libs/Python27.lib
-- pybind11 v2.2.1
-- Python Found and Enabled (with pybind11)

Then the more hidden part
You need to edit the C:\dev\Pangolin\src\CMakeLists.txt* file.

look for the text WIN , since the original CMake is ignoring python compile for windows platforms. you shold find something like the lines copied below.

option(BUILD_PANGOLIN_PYTHON "Build support for Pangolin Interactive Console" ON)
if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)
if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../external/pybind11/CMakeLists.txt")
add_subdirectory("../external/pybind11" "${CMAKE_CURRENT_BINARY_DIR}/external/pybind11")
set( pybind11_FOUND true)
else()
find_package(pybind11 QUIET)
endif()

delete the AND NOT WIN part from the if clause if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN), then save the file.

Finally, execute from *c:\dev\Pangolin\build*

cmake --build .

check the results, if you have warnigns ignore them, if you have error you need to solve them. In my case the error I got Fatal error LNK1104: cannot open file 'python27_d.lib, this is because CMake is compiling the project in debug mode, No clue why, I am no expert in CMAKE. so.

What I did was opening the C:\dev\Pangolin\build\pangolin.sln solution on Visual Studio 2017, change solution configuration form debug to release and recompiled within VS2017.

FINALLY I got all compiled successfully.

========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

I hope it helps, keep me posted if you need further asistance.

Regards

**Comment by Gatunox** [comment]
shuZro:

Do it as follows,

first create a folder with NO spaces on the name. in my case I have a c:\dev.
excute from cmd, you need to be admin, so make sure to excute it as an administrator.

within cmd position yourself in your folder, in my case c:\dev.
execute:

    git clone https://github.com/stevenlovegrove/Pangolin.git
    cd Pangolin

then if you plan to use Python exeute. make sure to be inside pangolin folder
execute:

    python -mpip install numpy pyopengl Pillow pybind11 **(These were OK)**
    git submodule init && git submodule update **(I noticed that once I delete all i had to donload it again)**

then execute:

    mkdir build
    cd build
    cmake ..

verify the results of CMake generation, and check for the lines showed bellow (I'm running python 2.7 btw), versions could change if you are under 3.7. The mots importat ones are the last two entries I am showing here.

    -- Found PythonInterp: C:/Python27/python.exe (found version "2.7.14")
    -- Found PythonLibs: C:/Python27/libs/Python27.lib
    -- pybind11 v2.2.1
    -- Python Found and Enabled (with pybind11)

Then the more hidden part
You need to edit the C:\dev\Pangolin\src\CMakeLists.txt* file.

look for the text WIN , since the original CMake is ignoring python compile for windows platforms. you shold find something like the lines copied below.

    option(BUILD_PANGOLIN_PYTHON "Build support for Pangolin Interactive Console" ON)
    if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)
    if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../external/pybind11/CMakeLists.txt")
    add_subdirectory("../external/pybind11" "${CMAKE_CURRENT_BINARY_DIR}/external/pybind11")
    set( pybind11_FOUND true)
    else()
    find_package(pybind11 QUIET)
    endif()

delete the AND NOT WIN part from the if clause if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN), then save the file.

Finally, execute from *c:\dev\Pangolin\build*

    cmake --build .

check the results, if you have warnigns ignore them, if you have error you need to solve them. In my case the error I got Fatal error LNK1104: cannot open file 'python27_d.lib, this is because CMake is compiling the project in debug mode, No clue why, I am no expert in CMAKE. so.

What I did was opening the C:\dev\Pangolin\build\pangolin.sln solution on Visual Studio 2017, change solution configuration form debug to release and recompiled within VS2017.

FINALLY I got all compiled successfully.

    ========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

I hope it helps, keep me posted if you need further asistance.

Regards

**Comment by Gatunox** [comment]
shuZro:

Do it as follows,

first create a folder with NO spaces on the name. in my case I have a c:\dev.
excute from cmd, you need to be admin, so make sure to excute it as an administrator.

within cmd position yourself in your folder, in my case c:\dev.
execute:

    git clone https://github.com/stevenlovegrove/Pangolin.git
    cd Pangolin

then if you plan to use Python exeute. make sure to be inside pangolin folder
execute:

    python -mpip install numpy pyopengl Pillow pybind11 **(These were OK)**
    git submodule init && git submodule update **(I noticed that once I delete all i had to donload it again)**

then execute:

    mkdir build
    cd build
    cmake ..

verify the results of CMake generation, and check for the lines showed bellow (I'm running python 2.7 btw), versions could change if you are under 3.7. The mots importat ones are the last two entries I am showing here.

    -- Found PythonInterp: C:/Python27/python.exe (found version "2.7.14")
    -- Found PythonLibs: C:/Python27/libs/Python27.lib
    -- pybind11 v2.2.1
    -- Python Found and Enabled (with pybind11)

Then the more hidden part
You need to edit the C:\dev\Pangolin\src\CMakeLists.txt* file.

look for the text WIN , since the original CMake is ignoring python compile for windows platforms. you shold find something like the lines copied below.

    option(BUILD_PANGOLIN_PYTHON "Build support for Pangolin Interactive Console" ON)
    if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)
    if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../external/pybind11/CMakeLists.txt")
    add_subdirectory("../external/pybind11" "${CMAKE_CURRENT_BINARY_DIR}/external/pybind11")
    set( pybind11_FOUND true)
    else()
    find_package(pybind11 QUIET)
    endif()

delete the AND NOT WIN part from the if clause if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN), then save the file.

Finally, execute from *c:\dev\Pangolin\build*

    cmake --build .

check the results, if you have warnigns ignore them, if you have error you need to solve them. In my case the error I got Fatal error LNK1104: cannot open file 'python27_d.lib, this is because CMake is compiling the project in debug mode, No clue why, I am no expert in CMAKE. so.

What I did was opening the C:\dev\Pangolin\build\pangolin.sln solution on Visual Studio 2017, change solution configuration form debug to release and recompiled within VS2017.

FINALLY I got all compiled successfully.

    ========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

I hope it helps, keep me posted if you need further asistance.

Regards

**Comment by shuZro** [comment]
Hey @Gatunox i am getting this when I run cmake ..

C:\Python35\mylibs\Pangolin\build>cmake ..
-- Building for: Visual Studio 15 2017
-- The C compiler identification is MSVC 19.14.26433.0
-- The CXX compiler identification is MSVC 19.14.26433.0
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.14.26428/bin/Hostx86/x86/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.14.26428/bin/Hostx86/x86/cl.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.14.26428/bin/Hostx86/x86/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.14.26428/bin/Hostx86/x86/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- MediaFoundation Found and Enabled
-- libpng Found and Enabled
-- libjpeg Found and Enabled
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Failed
-- Performing Test COMPILER_HAS_DEPRECATED
-- Performing Test COMPILER_HAS_DEPRECATED - Success
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Python35/mylibs/Pangolin/build

**Comment by Gatunox** [comment]
Hi,

I assume you have follow the reponse i've provided in the github site,
could you give the full output you are getting now?
In my case I am runnig the 32 bit version 2.7.14 of Python. Looking on your
previous response it seems you are under Python35, my guess this si 3.5
version

Let me know

Execute your python console and do the following.

import platform
platform.architecture()


Let me know
Cya

**Comment by shuZro** [comment]
@Gatunox I am so close! So I am only getting Fatal error LNK1104: cannot open file 'python35_d.lib

I did what you suggested and opened the sln in VS2017 switched to release and rebuilt the sln.

After doing so I just get a lot over errors... Not sure what to do.

Thanks

**Comment by shuZro** [comment]
I was able to fix the 64bit problem by running this cmake .. -G"Visual Studio 15 2017 Win64" btw

**Comment by Gatunox** [comment]
Have you tried to execute VS as Administrator? Close it and open it as adminitrator and try to compile again, It not clear the error, select one individual project and compile just one to see what's happening, Cheers

**Comment by shuZro** [comment]
Oh there we go :P Always run as Administrator! So my only question know is why cant i import in python...

**Comment by Gatunox** [comment]
Append these to the examples.

    import sys
    sys.path.append('../build/src/release')

Make rure that you have the pypangolin on that path **../build/src/release** this is realative on where you are running the actual example.

cya

**Comment by shuZro** [comment]
Aye thank you! Also do you just get black screens?

**Comment by Gatunox** [comment]
Yes, resize the window, and it will work, i'm trying to undertand why tha's happning, for sure its not an issue on the python version, I'm running the c++ example and it does the same weird behaviour, i will keep you posted if i find anything.

**Comment by shuZro** [comment]
Hey @Gatunox have you tried using pangolin.DrawPoints()? I get AttributeError: module 'pypangolin' has no attribute 'DrawPoints'

**Comment by shuZro** [comment]
@Gatunox So I was able to compile the other pangolin on windows for python. Let me know if you want the binding :)

**Comment by sasanasadiabadi** [comment]
@shuZro could you share how you resolved pangolin installation of windows for python?

**Comment by cbracketdash** [comment]
> shuZro:
> 
> Do it as follows,
> 
> first create a folder with **NO** spaces on the name. in my case I have a **c:\dev**.
> excute from **cmd**, you need to be **admin**, so make sure to excute it as an administrator.
> 
> within cmd position yourself in your folder, in my case **c:\dev**.
> execute:
> 
> ```
> git clone https://github.com/stevenlovegrove/Pangolin.git
> cd Pangolin
> ```
> 
> then if you plan to use **Python** exeute. **make sure to be inside pangolin folder**
> execute:
> 
> ```
> python -mpip install numpy pyopengl Pillow pybind11 **(These were OK)**
> git submodule init && git submodule update **(I noticed that once I delete all i had to donload it again)**
> ```
> 
> then execute:
> 
> ```
> mkdir build
> cd build
> cmake ..
> ```
> 
> verify the results of CMake generation, and check for the lines showed bellow (I'm running python 2.7 btw), versions could change if you are under 3.7. The mots importat ones are the last two entries I am showing here.
> 
> -- Found PythonInterp: C:/Python27/python.exe (found version "2.7.14")
> -- Found PythonLibs: C:/Python27/libs/Python27.lib
> -- pybind11 v2.2.1
> -- Python Found and Enabled (with pybind11)
> 
> Then the more hidden part
> You need to edit the **C:\dev\Pangolin\src\CMakeLists.txt*** file.
> 
> look for the text **WIN** , since the original CMake is ignoring python compile for windows platforms. you shold find something like the lines copied below.
> 
> ```
> option(BUILD_PANGOLIN_PYTHON "Build support for Pangolin Interactive Console" ON)
> if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)
> if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../external/pybind11/CMakeLists.txt")
> add_subdirectory("../external/pybind11" "${CMAKE_CURRENT_BINARY_DIR}/external/pybind11")
> set( pybind11_FOUND true)
> else()
> find_package(pybind11 QUIET)
> endif()
> ```
> 
> delete the **AND NOT WIN** part from the if clause **if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)**, then save the file.
> 
> Finally, execute from *_c:\dev\Pangolin\build*_
> 
> ```
> cmake --build .
> ```
> 
> check the results, if you have warnigns **ignore** them, if you have error you need to solve them. In my case the error I got **Fatal error LNK1104: cannot open file 'python27_d.lib**, this is because CMake is compiling the project in debug mode, No clue why, I am no expert in CMAKE. so.
> 
> What I did was opening the **C:\dev\Pangolin\build\pangolin.sln** solution on Visual Studio 2017, change solution configuration form **debug** to **release** and recompiled within VS2017.
> 
> FINALLY I got all compiled successfully.
> 
> ```
> ========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
> ```
> 
> I hope it helps, keep me posted if you need further asistance.
> 
> Regards

I did everything exactly as you said however everytime I type 'cmake .. ', it says "cmake is not recognized as an internal or external command, operable program or batch file.' Any ideas on what is wrong?

**Comment by hlin863** [comment]
> shuZro:
> 
> Do it as follows,
> 
> first create a folder with **NO** spaces on the name. in my case I have a **c:\dev**. excute from **cmd**, you need to be **admin**, so make sure to excute it as an administrator.
> 
> within cmd position yourself in your folder, in my case **c:\dev**. execute:
> 
> ```
> git clone https://github.com/stevenlovegrove/Pangolin.git
> cd Pangolin
> ```
> 
> then if you plan to use **Python** exeute. **make sure to be inside pangolin folder** execute:
> 
> ```
> python -mpip install numpy pyopengl Pillow pybind11 **(These were OK)**
> git submodule init && git submodule update **(I noticed that once I delete all i had to donload it again)**
> ```
> 
> then execute:
> 
> ```
> mkdir build
> cd build
> cmake ..
> ```
> 
> verify the results of CMake generation, and check for the lines showed bellow (I'm running python 2.7 btw), versions could change if you are under 3.7. The mots importat ones are the last two entries I am showing here.
> 
> -- Found PythonInterp: C:/Python27/python.exe (found version "2.7.14") -- Found PythonLibs: C:/Python27/libs/Python27.lib -- pybind11 v2.2.1 -- Python Found and Enabled (with pybind11)
> 
> Then the more hidden part You need to edit the **C:\dev\Pangolin\src\CMakeLists.txt*** file.
> 
> look for the text **WIN** , since the original CMake is ignoring python compile for windows platforms. you shold find something like the lines copied below.
> 
> ```
> option(BUILD_PANGOLIN_PYTHON "Build support for Pangolin Interactive Console" ON)
> if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)
> if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../external/pybind11/CMakeLists.txt")
> add_subdirectory("../external/pybind11" "${CMAKE_CURRENT_BINARY_DIR}/external/pybind11")
> set( pybind11_FOUND true)
> else()
> find_package(pybind11 QUIET)
> endif()
> ```
> 
> delete the **AND NOT WIN** part from the if clause **if(BUILD_PANGOLIN_PYTHON AND BUILD_PANGOLIN_GUI AND BUILD_PANGOLIN_VARS AND NOT WIN)**, then save the file.
> 
> Finally, execute from *_c:\dev\Pangolin\build*_
> 
> ```
> cmake --build .
> ```
> 
> check the results, if you have warnigns **ignore** them, if you have error you need to solve them. In my case the error I got **Fatal error LNK1104: cannot open file 'python27_d.lib**, this is because CMake is compiling the project in debug mode, No clue why, I am no expert in CMAKE. so.
> 
> What I did was opening the **C:\dev\Pangolin\build\pangolin.sln** solution on Visual Studio 2017, change solution configuration form **debug** to **release** and recompiled within VS2017.
> 
> FINALLY I got all compiled successfully.
> 
> ```
> ========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
> ```
> 
> I hope it helps, keep me posted if you need further asistance.
> 
> Regards

Thank you for providing a detailed outline for helping to set up Pangolin on Windows, when running the instruction "make .." in the build subfolder, I encountered an error message:
CMake Error at cmake/FindGLEW.cmake:51 (MESSAGE):
  Could not find GLEW
Call Stack (most recent call first):
  components/pango_opengl/CMakeLists.txt:46 (find_package)
  CMakeLists.txt:111 (include)


-- Configuring incomplete, errors occurred!

Is GLEW an additional package that I would need to install before building Pangolin on Windows?

Thank you and Kind regards,

**Comment by christian-rauch** [comment]
> Is GLEW an additional package that I would need to install before building Pangolin on Windows?

Yes. Please follow the build instructions. They reference the `install_prerequisites.sh` script that installs the build dependencies.

**Comment by wisnukusuma** [comment]
Hi @Gatunox I try your method and I got stuck to ** cmake ..** .
C:\Dev\Pangolin\build>cmake ..
-- Building for: Visual Studio 17 2022
-- Selecting Windows SDK version 10.0.22621.0 to target Windows 10.0.22631.
-- The C compiler identification is MSVC 19.37.32822.0
-- The CXX compiler identification is MSVC 19.37.32822.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.37.32822/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.37.32822/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Eigen: 'C:/Program Files/include/eigen3'
-- Found OpenGL: opengl32
CMake Error at cmake/FindGLEW.cmake:51 (MESSAGE):
  Could not find GLEW
Call Stack (most recent call first):
  components/pango_opengl/CMakeLists.txt:58 (find_package)
  CMakeLists.txt:111 (include)


-- Configuring incomplete, errors occurred!

**Comment by christian-rauch** [comment]
> Could not find GLEW

Did you install the dependencies according to the instructions via `install_prerequisites.sh`?

**Comment by wisnukusuma** [comment]
Yes but I can't install all dependencies in Windows, do you have any idea how to get this done in Windows?


> Did you install the dependencies according to the instructions via `install_prerequisites.sh`?

**Comment by christian-rauch** [comment]
> Yes but I can't install all dependencies in Windows, do you have any idea how to get this done in Windows?

Do you mean the `install_prerequisites.sh` script did not work or it worked without issues but you still cannot find GLEW?
